app/Http/Controllers/PermissionGroupController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Repository\PermissionGroupRepositoryInterface;
+use App\Http\Requests\Admin\PermissionGroupRequest;
+
+class PermissionGroupController extends Controller
+{
+    private $permissionGroupRepository;
+
+    public function __construct(PermissionGroupRepositoryInterface $permissionGroupRepository)
+    {
+        $this->permissionGroupRepository = $permissionGroupRepository;
+    }
+
+    public function index()
+    {
+        $permissionGroups = $this->permissionGroupRepository->paginate();
+        return view('admin.permission_group.index', ['permissionGroups' => $permissionGroups]);
```

Comment:
return view('admin.permission_group.index', ['permissionGroups' => $permissionGroups]);

=>

return view('admin.permission_group.index', [
      'permissionGroups' => $this->permissionGroupRepository->paginate()
]);